nemo/collections/common_parts/parts/multi_layer_perveptron.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Typo in file name - "multi_layer_perveptron.py"

Reply:
Aside of that: this one looks like a really **common** piece...

And btw. what do we do with parts in the **new NeMo**?